webui/react/src/pages/ExperimentList.settings.ts

```diff
@@ -195,8 +194,11 @@ const config: SettingsConfig<ExperimentListSettings> = {
       storageKey: 'tableOffset',
       type: number,
     },
+    user: {
+      defaultValue: undefined,
+      storageKey: 'user',
+      type: union([undefinedType, array(string)]),
+    },
   },
   storagePath: 'project-details',
```

Comment:
Although I know this is dead-code right now, I would also include the project id here since we have a ticket to change this.